mm2src/mm2_metrics/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "mm2_metrics"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+base64 = "0.10.0"
+common = { path = "../common" }
+derive_more = "0.99"
+futures = { version = "0.3", package = "futures", features = ["compat", "async-await", "thread-pool"] }
+itertools = "0.10"
+metrics = { version = "0.19" }
+metrics-util = { version = "0.13" }
+mm2_err_handle = { path = "../mm2_err_handle" }
+quanta = "0.9.3"
```

Comment:
This is used in tests only, and I even think this dep is not needed and can be replaced by standard library

Reply:
removed and refactored tests